pkg/macos/package.sh

```diff
@@ -248,31 +249,33 @@ else
     _failure
 fi
 
-
-if [ -z "${NIGHTLY}" ]; then
+SRC_PKG = "$SCRIPT_DIR/salt-src-$VERSION-py3-$CPU_ARCH.pkg"
+if [ "${SIGN}" -eq 1 ]; then
     _msg "Building the product package (signed)"
     # This is not a nightly build, so we want to sign it
-    FILE="$SCRIPT_DIR/salt-$VERSION-py3-$CPU_ARCH-signed.pkg"
+    DST_PKG="$SCRIPT_DIR/salt-$VERSION-py3-$CPU_ARCH-signed.pkg"
     if productbuild --resources="$SCRIPT_DIR/pkg-resources" \
                     --distribution="$DIST_XML" \
-                    --package-path="$SCRIPT_DIR/salt-src-$VERSION-py3-$CPU_ARCH.pkg" \
+                    --package-path="$SRC_PKG" \
                     --version="$VERSION" \
                     --sign "$DEV_INSTALL_CERT" \
                     --timestamp \
-                    "$FILE" > "$CMD_OUTPUT" 2>&1; then
+                    "$DST_PKG" > "$CMD_OUTPUT" 2>&1; then
+        echo "Renaming $DST_PKG -> $SRC_PKG"
```

Comment:
This isn't the right name... we don't want `-src-` in the name. The `-src-` package is different than the signed package.

Reply:
ah, yes

Reply:
Should be fixed now